orders/tests.py

```diff
@@ -0,0 +1,61 @@
+from django.utils import timezone
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+from .models import Order
+from .models import StatusOptions
+from django.core.exceptions import ObjectDoesNotExist
+import pytest
+
+STARTING_DATE = timezone.now()
+FINISHING_DATE = timezone.now()
+TOTAL_PRICE = 100
+STATUS = StatusOptions.Pending
+
+
+@pytest.fixture
+def order():
+    return Order.create(dog_owner_id=DogOwner.objects.get(id=1),
+                        daycare_id=DayCare.objects.get(id=1),
+                        starting_date=STARTING_DATE,
+                        finishing_date=FINISHING_DATE,
+                        total_price=TOTAL_PRICE,
+                        )
+
+
+@pytest.mark.django_db
+class TestOrderModel:
+    def test_create_order(self):
+        new_order = Order.create(dog_owner_id=DogOwner.objects.get(id=1),
+                                 daycare_id=DayCare.objects.get(id=1),
+                                 starting_date=STARTING_DATE,
+                                 finishing_date=FINISHING_DATE,
+                                 total_price=TOTAL_PRICE)
+        assert new_order in Order.objects.all()
+
+    def test_delete_order(self, order):
+        assert order in Order.objects.all()
+        order.delete()
+        assert order not in Order.objects.all()
+
+    def test_get_not_existed_order_by_order_id(self):
+        try:
+            Order.objects.get(id=-1)
+            assert False
+        except ObjectDoesNotExist:
+            assert True
```

Comment:
This test is not mandatory, it actually tests Django 'get' internal mechanism.
We take for granted that using a model .object.get() will work, its basic functionally of the framework
if it doesnt work the framework is broken.

Reply:
Got it. Removing it.